demo/src/index.js

```diff
@@ -30,6 +31,7 @@ const content = [
     text: "We shall never surrender...",
     className: "classE",
     animation: "zoom",
+    link: "https://example.org/",
```

Comment:
Minor: I suggest you use a different URL here for distinguishing from the previous one.
https://google.com for instance.

Reply:
Clearer that the example.com and example.org I was using - thanks